<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>640</width>
                <height>480</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>640</width>
                <height>480</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>640</width>
                <height>480</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Preference</string>
        </property>
        <widget class="QPushButton" name="buttonOK">
            <property name="geometry">
                <rect>
                    <x>360</x>
                    <y>440</y>
                    <width>80</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="text">
                <string>OK</string>
            </property>
        </widget>
        <widget class="QPushButton" name="buttonCancel">
            <property name="geometry">
                <rect>
                    <x>450</x>
                    <y>440</y>
                    <width>80</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="text">
                <string>Cancel</string>
            </property>
        </widget>
        <widget class="QPushButton" name="buttonApply">
            <property name="geometry">
                <rect>
                    <x>540</x>
                    <y>440</y>
                    <width>80</width>
                    <height>25</height>
                </rect>
            </property>
            <property name="text">
                <string>Apply</string>
            </property>
        </widget>
        <widget class="QGroupBox" name="groupBox">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>620</width>
                    <height>80</height>
                </rect>
            </property>
            <property name="title">
                <string>Fractal Config</string>
            </property>
            <widget class="QComboBox" name="comboFractalType">
                <property name="geometry">
                    <rect>
                        <x>111</x>
                        <y>43</y>
                        <width>140</width>
                        <height>22</height>
                    </rect>
                </property>
                <item>
                    <property name="text">
                        <string>Mandelbrot Set</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>Julia Set</string>
                    </property>
                </item>
            </widget>
            <widget class="QLabel" name="label">
                <property name="geometry">
                    <rect>
                        <x>18</x>
                        <y>42</y>
                        <width>100</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Fractal Type:</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_2">
                <property name="geometry">
                    <rect>
                        <x>266</x>
                        <y>42</y>
                        <width>200</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Julia Set Constant(a + bi):</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="editJuliaC">
                <property name="geometry">
                    <rect>
                        <x>446</x>
                        <y>43</y>
                        <width>157</width>
                        <height>22</height>
                    </rect>
                </property>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
